### PR TITLE
Add example code #1

### PR DESCRIPTION
**Pull request by jiro4989**

### CODE REVIEW

**Comment by SolitudeSF** [comment]
thanks, thats pretty cool. but i would like you to flatten the directory structure a bit. no need for separate folder for each example. just put `flip/resize.nim` in examples folder.

**Comment by jiro4989** [comment]
@SolitudeSF thanks, I fixed.

**Comment by jiro4989** [comment]
Ah, sorry, i will fix more.

**Comment by SolitudeSF** [comment]
also, resulting images are not needed. people who want to see result can just run the example. also i dont want to depend on stability of results until they are covered by tests (in case i find an error in algorithm). leave just the sample image. and please, squash the commits into one afterwards.

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,5 @@ nimcache
 !tests/image.jpg
 tests/t*
 !tests/t*.nim
+examples/*
+!examples/*.*
```
Comment (SolitudeSF) [inline .gitignore]: this gonna unignore `exe`'s on windows. better match `.nim`/`.nims` and sample file explicitly

Comment (jiro4989) [reply .gitignore]: thanks, I fixed

Comment (jiro4989) [reply .gitignore]: Ah, sorry, i fix more

Comment (jiro4989) [reply .gitignore]: I changed to

```
examples/*
!examples/*.nim
!examples/*.nims
!examples/sample.png
```

**Comment by SolitudeSF** [comment]
thanks!

**Pull request merged**